blackJack.js

```diff
@@ -0,0 +1,113 @@
+document.addEventListener('DOMContentLoaded', () => {
+    let total = 0
+    let res = 0
+    let newArr =[]
+    const newDeckCards = async () => {
+        try {
+            let newDeck = await axios.get("https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1")
+            let id = document.querySelector("#deckId")
+            id.innerText = newDeck.data.deck_id
```

Comment:
you can declare your deck_id variable globally and set it to zero like you are doing it with other variables, or just not set it to any value. Just "let deck_id;" . 
Later you will grab the deck id after making the axios call and update the variable to the value of "newDeck.data.deck_id" (instead of updating the inner text of the div in your html file).